src/engine/zone.h

```diff
@@ -109,6 +109,7 @@ struct Zone : MoveableOnly<Zone>, HasGroupZoneProcessors<Zone>, SampleRateSuppor
         bool active{false};
         SampleID sampleID;
         int64_t startSample{-1}, endSample{-1}, startLoop{-1}, endLoop{-1};
+        dsp::InterpolationTypes interpolationType{dsp::InterpolationTypes::Sinc};
```

Comment:
Minor: could use `auto` here, for brevity.

Reply:
no you can't. that's a non-static struct member not a local variable.